### ISSUE TITLE
enhancement: new gcode commands for resetting material properties

### ISSUE THREAD

**Issue by Dim3nsioneer** [daid/LegacyCura#886]
@daid: What do think about the idea of implementing one or more commands (M6xx?) for resetting UM2 settings to the values set at the start of a print? This might be quite helpful for using with Cura plugins, e.g. TweakAtZ.

**Comment by daid** [comment]
Yes, bunch of these need to be implemented. Also if we want to improve for example OctoPrint printing on the UM2.

**Comment by Dim3nsioneer** [comment]
Do you have this on your todo list?
